features/weth/Weth.tsx

```diff
@@ -81,7 +81,7 @@ const Weth = () => {
     wethBalance,
     ethBalance,
   } = WethContract.useContainer();
-  const { signer } = Connection.useContainer();
+  const { signer, notify } = Connection.useContainer();
```

Comment:
This file doesn't look like its merged in latest with master, its prob worth rebasing now and fix some conflicts

Reply:
done and updated PR to use master